### ISSUE TITLE
Search via Plebnames, lookup npub in Bitcoin OP_RETURN

### ISSUE THREAD

**Issue by MichiSpebach** [v0l/snort#586]
[enhancement, question]:
When I implement searching npubs via Plebnames like showcased at http://plebnames.org/?url=test would this pull request be merged?
Plebnames would provide completely decentralized, uncensorable and unique aliases for npubs. Only normal Bitcoin explorers like mempool.space or blockstream.info need to be accessed.

**Describe the solution you'd like**
When I search for a name in the top right search bar a lookup in the Bitcoin chain is made (via. arbitrary Bitcoin explorers) to check if there is a Plebname with an attached Nostr npub. If there is one it appears in the search results.

**Comment by v0l** [comment]
Could be a simple addition but it wont have much usage

**Comment by MichiSpebach** [comment]
Yeah, I agree for now, it is based on the network effect, but I encourage everyone to look into the Plebnames protocol at https://github.com/MichiSpebach/plebnames and claim your name with just your Bitcoin wallet. Would be cool to have a truly decentralized and uncensorable name system to lookup npubs.

I start now with understanding the snort codebase and finding the correct place to add this to the search. Any hint where to look?

**Comment by v0l** [comment]
It should go here: https://git.v0l.io/Kieran/snort/src/commit/a3b53de456394f27a243d5a257c78187a81ce6e8/packages/app/src/Hooks/useLoginHandler.tsx

**Comment by MichiSpebach** [comment]
Thanks for the hint. If I am not mistaken this is about logging in, I want to build it into the search. Or is logging in with a npub the same? Anyway I will find it.

**Comment by MichiSpebach** [comment]
implemented a first draft, let me know what you think about it.

**Comment by ndee** [comment]
`plebnames` is such an elegant solution for a decentralized and efficient name service - on `Bitcoin`. I hope this PR gets merged.
